server/deeplinks/deeplinks.go

```diff
@@ -104,25 +93,34 @@ func EvaluateDeepLinksResponse(obj map[string]interface{}, name string, links []
 			switch resOut := out.(type) {
 			case bool:
 				if resOut {
-					finalLinks = append(finalLinks, &application.LinkInfo{
-						Title:       ptr.To(link.Title),
-						Url:         ptr.To(finalURL.String()),
-						Description: link.Description,
-						IconClass:   link.IconClass,
-					})
+					continue
```

Comment:
should i ` if !isValidURL(link.URL) {
            errors = append(errors, fmt.Sprintf("invalid URL: %v for resource %v", link.URL, name))
            continue
        } ` add this before `continue` where `func isValidURL(u string) bool {
    _, err := url.ParseRequestURI(u)
    return err == nil
}` so that before going to next iteration. If the URL is invalid (`isValidURL` returns `false`), an error message is appended to the `errors` slice, and the `continue` statement is executed, skipping the rest of the current iteration and moving to the next iteration.